src/ui/details/list_item.rs

```diff
@@ -284,7 +286,7 @@ pub fn list_item(param: ListItem) -> impl View {
 						field_value.set(pass.clone());
```

Comment:
Currently when the field saves it shows the contents of pass, this line is why. The following would fix that. 
I don't think it needs a check to see if the field is secret as only secret fields get the generator. 

`field_value.set(String::from(SECRET_MULTILINE_PLACEHOLDER));`

Reply:
I'm not 100% sure I understand. That line makes sure that when we have a single line field it shows the new generated secret after the generator is done. If I replaced it with your line two things would happen:
1. The field would show the placeholder in edit mode
2. All single line fields will show the multi line placeholder

Line by line here:
```rust
// get the password from our generator
let mut pass = generate_password(entropy);

// set single line field to that password (will be ignored if this is a multi line field)
field_value.set(pass.clone());

// append the password to the multi line field (will be ignored if this is a single line field)
field_doc.get().edit_single(
	Selection::region(0, field_doc.get().text().len()),
	&format!("{}\n{}\n", field_doc.get().text(), pass),
	EditType::DeleteSelection,
);
```

Reply:
Ahhh I found the bug you're talking about. Yes when you edit a multi line field and use the generator to add a secret and then save it shows the secret... that's bad. Not sure why the edit_button_slot doesn't reset the value to the `SECRET_MULTILINE_PLACEHOLDER` value properly...

Reply:
I found the bug! The `save_edit` function reset only the multi line field:

https://github.com/dominikwilkowski/vault/blob/06e917a4f7ed75821e1c291cf021b077dc5fe456/src/ui/details/detail_view.rs#L95-L101

... and not the `field_value` because I thought like why would I need the field value... of course I need it for when we hide the editor in normal mode ... duh!
Committing the fix tomorrow. Thanks for finding this! Totally missed this bug

Reply:
fixed via 51b726bda25c67fef7868a008f40ada8d5326b02